Wireless.py

```diff
@@ -50,8 +50,11 @@ def __init__(self):
     # ======================== public ==========================================
     
     def indicateDevices(self,devices):
-        self.devices         =  devices
-    
+        self.devices         = devices
+
+    def overridePDR(self,pdr):
+        self.PDR             = pdr
```

Comment:
you cannot change the value of an all-caps "variable" by convention. Please create an attribute called `constantPDR`